lib/proxy.js

```diff
@@ -771,4 +778,18 @@ function setHttp(link) {
   return link;
 }
 
+
+// To allow CORS
+
+function allowCORS(objServer, opts){
+    if(_.isEmpty(opts['cors']) === false && opts['cors']['allow'] && opts['cors']['allow'] === true){
+        objServer.use(function(req, res, next) {
+            res.header("Access-Control-Allow-Origin", "*");
```

Comment:
redbird is not implemented using express, so ```.use``` is not available. This would need to be implemented directly on nodejs http object.